content/en/docs/setup/upgrade/canary/index.md

```diff
@@ -56,6 +56,14 @@ istio-sidecar-injector          1          7m56s
 istio-sidecar-injector-canary   1          3m18s
 {{< /text >}}
 
+{{< warning >}} 
+Due to [a bug](https://github.com/istio/istio/issues/28880) in the creation of `ValidatingWebhookConfiguration`s during install, initial installations of Istio __must not__ not specify a revision. As a temporary workaround, for Istio resource validation to continue working after removing the non-revisioned Istio installation, the `ValidatingWebhookConfiguration` must be manually pointed to the revision that should handle validation.
+
+One way to accomplish this is to edit the webhook in-place by running `kubectl edit validatingwebhookconfiguration istiod-istio-system` and changing the `service` name from `istiod` to `istiod-<REVISION>` where `<REVISION>` is the revision that should handle validation. Alternatively, a `ValidatingWebhookConfiguration` can be created from [this template](https://raw.githubusercontent.com/istio/istio/master/manifests/charts/base/templates/validatingwebhookconfiguration.yaml) and applied after removing the old validating webhook.
+
+An alternative to changing where the `ValidatingWebhookConfiguration` points is to manually create a service called `istiod` pointing to the target revision using [this service](https://github.com/istio/istio/blob/master/manifests/charts/istio-control/istio-discovery/templates/service.yaml) as a template.
```

Comment:
Should we mention the implications of this? Is this only for validation or can the user use this as a hacky version of a default tag?

Reply:
This would only work for validation, but if we added a `MutatingWebhook` to base pointed to `istiod` this could work as a hacky default tag as well

Reply:
I'd rather we do the real default tag support so this is fine as is, thanks.